### ISSUE TITLE
parser errors out when parse "separator ';'"

### ISSUE THREAD

**Issue by camoonke** [taozhi8833998/node-sql-parser#1249]
[uppercase "SEPARATOR" is fixed](https://github.com/taozhi8833998/node-sql-parser/issues/1234)

however, there exists a problem when using lowercase "separator".

**- Describe the bug**
parser errors out when parse "separator ';'"

**- Database Engine**
Mysql

To Reproduce
**-the SQL that be parsed**
`select GROUP_CONCAT(DISTINCT abc order by abc) as abc from business_table`

**- version**
node-sql-parser: 4.6.3
node version: v10.20.0

**code**
```js
const { Parser } = require('node-sql-parser');
const parser = new Parser();
const sql = `select GROUP_CONCAT(DISTINCT abc order by abc separator ';')`; 
const ast = parser.astify(sql);
```

**error**
Expected "!=", "#", "%", "&", "&&", "'", "(", ")", "*", "+", ",", "-", "--", "->", "->>", ".", "/", "/*", "0x", "<", "<<", "<=", "<>", "=", ">", ">=", ">>", "ASC", "BETWEEN", "DESC", "IN", "IS", "LIKE", "NOT", "REGEXP", "RLIKE", "SEPARATOR", "X", "XOR", "\"", "^", "_binary", "b", "div", "|", "||", "~", or [ \t\n\r] but "s" found.

**Comment by taozhi8833998** [comment]
@camoonke Sorry for the case sensitive.

**Comment by camoonke** [comment]
thanks a lot.

**Comment by taozhi8833998** [comment]
@camoonke fixed in the new version `4.6.4`
